napi-inl.h

```diff
@@ -898,47 +898,65 @@ inline const T* TypedArray_<T,A>::Data() const {
 // Function class
 ////////////////////////////////////////////////////////////////////////////////
 
-inline Function Function::New(napi_env env,
-                              VoidFunctionCallback cb,
-                              const char* utf8name,
-                              void* data) {
-  // TODO: Delete when the function is destroyed
-  VoidFunctionCallbackData* callbackData = new VoidFunctionCallbackData({ cb, data });
+// Helpers to handle functions exposed from C++.
+namespace helper {
```

Comment:
Yeah, the explicit template specialization doesn’t work outside of namespace scope 😩